tests/test_config.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+
+from flask import current_app
+from flask_testing import TestCase
+
+from bucketlist import app
+
+
+class TestDevelopmentConfig(TestCase):
```

Comment:
hmmm @RyanSept In this file she is testing the 3 config classes that they are configured correctly..